libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -935,20 +936,20 @@ + (NSString *)advancedAuthStateDesc:(SFOAuthAdvancedAuthState)authState
 
 - (NSURLSession*)session {
     if (_session == nil) {
-        _session = [NSURLSession sessionWithConfiguration:[NSURLSessionConfiguration ephemeralSessionConfiguration]];
+        SFNetwork *network = [[SFNetwork alloc] init];
+        _session = network.activeSession;
```

Comment:
Is there value to doing this here, if it's not actually going to make use of `SFNetwork` functionality?